src/App.js

```diff
@@ -1,11 +1,236 @@
+                <div className="personalData">
+                    <label htmlFor="first-name-field">Voornaam
+                        <input
+                            type="text"
+                            id="first-name-field"
+                            name="firstName"
+                            ref={register({ required: true})}
+                        />
+                        {errors.firstName && <span className="error-message">Verplicht invullen</span>}
+                    </label>
+                    <label htmlFor="last-name-field">Achternaam
+                        <input
+                            type="text"
+                            id="last-name-field"
+                            name="lastName"
+                            ref={register({ required: true})}
+                        />
+                        {errors.lastName && <span className="error-message">Verplicht invullen</span>}
+                    </label>
+                    <label htmlFor="age-field"> Leeftijd
+                        <input
+                            type="text"
+                            id="age-field"
+                            name="age"
+                            ref={register({ required: true, min: 18})}
+                        />
+                        {errors.age && <span className="error-message">Je moet minimaal 18 jaar zijn</span>}
+                    </label>
+                    <label htmlFor="zipcode-field"> Postcode
+                        <input
+                            type="text"
+                            id="zipcode-field"
+                            name="zipcode"
+                            ref={register({ required: true, pattern: /^(?:NL-)?(\d{4})\s*([A-Z]{2})$/i})}
+                        />
+                        {errors.zipcode && <span className="error-message">Vul volgens het juiste format in: 1234AA</span>}
+                    </label>
+                    <label htmlFor="house-number-field"> Huisnummer
+                        <input
+                            type="text"
+                            id="house-number-field"
+                            name="houseNumber"
+                            ref={register(
+                                { required: true})}
+                        />
+                        {errors.houseNumber && <span className="error-message">Verplicht invullen</span>}
+                    </label>
+                </div>
+
+                <div className="deliveryTime">
+                    <p>Bezorgfrequentie</p>
+                    <label htmlFor="everyWeek">
+                        <input
+                            type="radio"
+                            name="delivery"
+                            id="everyWeek"
+                            value="weekly"
+                            ref={register({ required: true})}
+                        />Iedere week
+                    </label>
```

Comment:
Ook dit had mooi gestaan in een los component. Je moet eigenlijk altijd voor jezelf op zoek gaan naar stukjes code wat zich meerdere keren herhaalt.